src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/CaseloadItem.kt

```diff
@@ -37,7 +38,7 @@ data class CvlFields(
 @Schema(description = "An item in the caseload")
 data class CaseloadItem(
   @Schema(description = "Details about a prisoner")
-  val prisoner: Prisoner,
+  val prisoner: PrisonerSearchPrisoner,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Is this used anywhere else which may cause an unintended problem somewhere else (presumably not as everything is passing)?

Reply:
Ya it was used in one place and I have fixed it as well.

Reply:
fixed